cpp/core.cc

```diff
@@ -164,6 +165,9 @@ List<PasswdEntry*>* GetAllUsers() {
   endpwent();
 
   return ret;
+  #else
+  throw Alloc<RuntimeError>(StrFromC("Compiled without gepwent support. user completion not possible"));
```

Comment:
I think this might cause a crash?

Although it's a little cheesy, I think it could be OK to return the empty list and print a warning to stderr about HAVE_PWENT?

So it won't crash but there will be some spam

I'm open to other ideas